relm4/src/component/sender.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2022 System76 <info@system76.com>
+// SPDX-License-Identifier: MIT or Apache-2.0
+
+use crate::{Component, Sender};
+use std::rc::Rc;
+
+/// Contain senders used by the component.
+pub type ComponentSender<C> = Rc<ComponentSenderInner<C>>;
```

Comment:
I don't think `Rc` is a good choice here, I'd use `Arc` instead. You might want to send messages from your own background thread or use commands as actors in a threaded runtime.

Reply:
Sure thing